stts/Services/StatusPage/Packet.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Packet.swift
+//  stts
+//
+//  Created by D on 1/6/17.
+//  Copyright © 2017 inket. All rights reserved.
+//
+
+import Foundation
+
+class Packet: StatusPageService {
+    override var name: String { return "Packet" }
```

Comment:
Feel free to remove the `name` line. The name is now inferred from the class name!